dsgrid/cli/registry.py

```diff
@@ -930,7 +1135,10 @@ def list_datasets(registry_manager, filter):
 @click.pass_obj
 @click.pass_context
 def register_dataset(ctx, registry_manager, dataset_config_file, dataset_path, log_message):
-    """Register a new dataset with the dsgrid repository."""
+    """Register a new dataset with the registry. The JSON/JSON5
+    filename must match the data model defined by this documentation:
```

Comment:
file contents? Or revert to language like, "Must mach the data model defined by"